scenarios/csharp/dotnetcore/call-center/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "YourSubscriptionKey": "cd281650d4ba4063bee7edf4a24ce7f7",
```

Comment:
Please don't check in any Secret!!!!!

Reply:
Please remember to revert this token for the azure service since this is a public repo and everyone gets this token now.

Reply:
Sorry. I'll revert the resources key.

Reply:
The secrets has been regenerated